### ISSUE TITLE
Lat / Long Support?

### ISSUE THREAD

**Issue by aunyks** [kochrt/timeline-maker#112]
Does Markwhen support using geographic coordinates (latitude & longitude) for locations? I tried plugging in a couple for a location but they didn’t seem to show up on the map.

**Comment by kochrt** [comment]
Yeah just tried it myself, doesn't seem to work. Admittedly I won't get to it soon but I will leave this open so it's on my radar

**Comment by aunyks** [comment]
Cool! Thanks for acknowledging, awesome project btw.
